docker-compose.test.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  # Application
+  test:
+    image: nhsei:latest
+    environment:
+      - DJANGO_SECRET_KEY=entersecretkeyhere
+      - SERVER_ENV=test
+    restart: always
+    build:
+      context: ./
+      target: app
+      args:
+        - SERVER_ENV=test
+    volumes:
+      - ./:/usr/srv/app:Z
+      - ./media:/usr/srv/app/media:Z
+    command:
+      - /bin/bash
+      - -c
+      - |
+        black --check .
```

Comment:
Is it worth format checking at the end so that a failure still runs the tests first? Especially if we don't have an automated hook to run it laptop-side.

Reply:
I think if we're checking code style we can fail fast on that, rather than spending time running a full test suite only for a late (and quickly fixed) syntax failure to trip it up.